### PR TITLE
Afform GUI: add bootstrap3

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
Add the Bootstrap3 bundle to Afform base pages, plus a little code cleanup.

Before
----------------------------------------
No Bootstrap theme.

After
----------------------------------------
Looks much better.

Comments
----------------------------------------
As I mentioned in a `TODO` comment, we could make this more flexible by allowing each afform to declare its own theming requirements.

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
ok - this is the same as I did for search so all good - what about CLE?

File: ext/afform/gui/xml/Menu/afform_gui.xml [hunk 1]
```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0"?>
```
Comment (eileenmcnaughton) [inline ext/afform/gui/xml/Menu/afform_gui.xml:1]: Why did this just get removed?

Comment (colemanw) [reply ext/afform/gui/xml/Menu/afform_gui.xml:1]: Per the code comment I added, the path `civicrm/admin/afform` doesn't need to be registered in Civi's menu as it's registered as an afform and handled via afform's magical routing.
We made that switch a while ago, this is just cruft.

**Comment by colemanw** [comment]
> ok - this is the same as I did for search so all good - what about CLE?

I can add it to CLE but it won't work for another 2 months when 5.31 is released.

**Comment by eileenmcnaughton** [comment]
@colemanw re CLE - it won't break anything will it?

**Comment by colemanw** [comment]
> @colemanw re CLE - it won't break anything will it?

Well yea it will hard crash because the method `addBundle` is not defined prior to 5.31. I'll just have to bump up the core req.

**Comment by eileenmcnaughton** [comment]
@colemanw that's fine - there's still a working version for earlier civi versions

**Comment by colemanw** [comment]
@eileenmcnaughton https://github.com/civicrm/org.civicrm.contactlayout/pull/82

**Pull request merged**